uk.co.compucorp.civicrm.hrcore/gulp/tasks/backstopjs.js

```diff
@@ -118,41 +119,45 @@ function createTempConfigFile () {
 }
 
 /**
- * Fetches civicrm contacts whose emails match "civihr_" pattern
- * and returns a map of their "roles" connected to their IDs.
- * Requires 'civihr_(staff|manager|admin)@...' to be presented in DB,
- * otherwise will throw an error.
+ * Given a set of UF matches, it finds the contact with the specified drupal id
  *
- * @return {Promise} resolved with {Object}, ex. { 'staff': 204, ... etc }
+ * @param {Array} ufMatches
```

Comment:
Is `ufMatch` a specific term for Civi/Drupal? I tried to google it and cannot find an explanation. Please explain what it is

Reply:
It's the name of the Civi API entity `UFMatch`